src/style.css

```diff
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-position: center;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
combine selectors to not repeat code

Reply:
I don't understand how to do it yet. Given the conditions.